web/themes/custom/usagov/templates/field--node--body--wizard-step.html.twig

```diff
@@ -24,7 +24,7 @@
         {% elseif element["#language"] == "es" %}
         <div class="grid-row flex-justify-center maxw-mobile margin-x-auto margin-top-6">
             <a class="usa-button" href="/es/donde-reportar-una-estafa/que-tipo-de-estafa-necesita-reportar#block-usagov-content">
-                comience desde el principio
+                Reiniciar
```

Comment:
Is this an intentional change? I'm going to assume it is, but if not, there'll be time to make another change.